common/src/main/java/com/wynntils/mc/mixin/LocalPlayerMixin.java

```diff
@@ -29,4 +31,11 @@ private void redirectChat(ClientPacketListener connection, Packet<?> packet, Str
 
         connection.send(new ServerboundChatPacket(result.getMessage()));
     }
+
+    @Inject(method = "drop", at = @At("HEAD"), cancellable = true)
+    private void onDropPre(boolean fullStack, CallbackInfoReturnable<Boolean> cir) {
```

Comment:
Could this not be just a normal CallbackInfo?

Reply:
MC development plugin suggests that (as it should match method return).

Reply:
And drop returns boolean.